Library/Formula/sslsplit.rb

```diff
@@ -0,0 +1,38 @@
+class Sslsplit < Formula
+  homepage "https://www.roe.ch/SSLsplit"
+  url "https://github.com/droe/sslsplit/archive/0.4.11.tar.gz"
+  sha256 "a33808f5d78000d1a919e61bbdfc48478ad40c1a9b09a0250b322f8954cc5ae0"
+
+  depends_on "check" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libevent"
+  depends_on "openssl"
+
+  # Patch to remove UID 0,GID 0 setting for install
+  patch :DATA
+
+  def install
+    ENV.deparallelize
+    system "make", "PREFIX=#{prefix}", "test", "install"
+  end
+
+  test do
+    system "#{bin}/sslsplit", "-V"
```

Comment:
Nope, we want a runtime test based on the installed files e.g. from the homepage `sslsplit -k ca.key -c ca.pem -P  https 127.0.0.1 8443  https ::1 8443`